browser/src/robocorp/browser/__init__.py

```diff
@@ -24,54 +23,50 @@
 
 
 def configure(**kwargs) -> None:
-    """
-    May be called before any other method to configure the browser settings.
-
-    Calling this method is optional (if not called a default configuration will
-    be used -- note that calling this method after the browser is already
-    initialized will have no effect).
+    """Configures browser settings before any other method is called.
 
+    This method is optional, and if not invoked, default configurations will be used.
+    Note that calling this method after the browser is initialized has no effect.
+    
+    Example:
+        ```python
+        browser.configure(browser_engine="firefox", slowmo=100)
+        ```
+    
     Args:
-        browser_engine:
-            Browser engine which should be used
+        browser_engine: Browser engine which should be used
             default="chromium"
             choices=["chromium", "chrome", "chrome-beta", "msedge",
                      "msedge-beta", "msedge-dev", "firefox", "webkit"]
```

Comment:
Even if we split these on multiple lines, won't they get rendered as separated by spaces? Like:

> Browser engine ... used default="chromium" choices=["chromium", ...

If that happens, we'll need some extra characters/paranthesis to separate these better.

Reply:
Agreed, will try to rephrase this